tests/by-util/test_df.rs

```diff
@@ -147,11 +147,12 @@ fn test_use_percentage() {
 
     for line in lines {
         let mut iter = line.split_whitespace();
+        // Skip "Filesystem" column
```

Comment:
Now that the `--output` argument works, let's update this test to just use `--output=used,avail,pcent` so that we don't have to do so much parsing and skipping.

Reply:
Good point, thanks.